Packs/IllusiveNetworks/Playbooks/playbook-Illusive-Incident-Escalation.yml

```diff
@@ -0,0 +1,1482 @@
+id: Illusive-Incident-Escalation
+version: -1
+name: Illusive-Incident-Escalation
+description: This playbook is used for creating an automatic analysis of the Illusive’s
```

Comment:
Please replace all of the places where this character is found -> ’ with ' (it's failing when trying to upload to the server)